cmd/panicmonitor/go.mod

```diff
@@ -0,0 +1,8 @@
+module panicmonitor
```

Comment:
go.mod should be placed at the root of the project, and the module name should be `github.com/lujjjh/panicmonitor`.